plugin/src/test/kotlin/com/github/triplet/gradle/play/CompatibilityTest.kt

```diff
@@ -110,9 +101,9 @@ class CompatibilityTest(
         @JvmStatic
         @Parameterized.Parameters(name = "agpVersion: {0}, gradleVersion {1}")
         fun parameters() = listOf(
-                arrayOf("3.0.1", "4.1"), // Oldest supported
+                arrayOf("3.1.4", "4.4"), // Oldest supported
```

Comment:
Can we go down to `3.1.0`? Our `CompatibilityTest` passes with if we also set MIN_AGP_VERSION to 3.1.0. I would like to minimize the impact of this PR in terms of backwards compatibility